src/routes/chat/ChatInput/ChatInput.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import SendIcon from '../../../assets/images/send_icon.png';
```

Comment:
use absolute paths. `import SendIcon from 'assets/.....';`.
In fact, why do we need this image? Use fontawesome or something: https://fontawesome.com/icons?d=gallery&q=send

Always prefer SVGs over images.